source/chapters/djing_with_mixxx.rst

```diff
@@ -303,6 +303,51 @@ and click the :guilabel:`Play` button next to the waveform.
 To display the Preview deck, press :kbd:`CTRL` + :kbd:`4` (Windows/Linux) or
 :kbd:`CMD` + :kbd:`4` (Mac).
 
+.. _djing-gain-staging:
+
+Setting your levels properly (gain staging)
+===========================================
+.. sectionauthor::
+   Be <be.0@gmx.com>
+
+Setting your levels properly, also known as gain staging, is essential for
+getting the best sound quality out of the equipment you are using. At every link
+in your signal chain, from Mixxx's channel gains to the power amplifier, the level
+should be well above the noise floor, but lower than the maximum level
+before the signal clips. The :term:`level meters <level meter>` should mostly be
+around the top of their green region. The average level should not be in the
+yellow region. Use the yellow region to leave headroom, or available level
+above the average before the signal clips. The loudest parts of the music
+(the transients) should briefly go into the yellow region.
+
+**If a level meter is in its red region, the signal is clipping and the gain should be
+turned down.** Some equipment doesn't have a level meter and only has an LED that
+turns on when the signal clips. Clipping means that the peaks of the waveform are
+flattened because the equipment has reached the maximum level that it can amplify
+the signal to. This distorts sound in an unpleasant way and can damage equipment
+by driving it with more power than it is designed to handle. Increasing
+the gain past the point where clipping begins (further into the red on a meter)
+will distort the signal more. If there is a limiter being used with the sound
+system (which is usually the case on large PA systems), clipping more will not
+make the sound any louder but will distort it more.
+
+On the other hand, the signal should not be too low because every audio device
+generates a little noise. The level of this noise is called the noise floor. It
+does not increase when the device is turned up; only the signal does. So,
+for the best signal-to-noise ratio (SNR), the signal should be strong. Every
+time the signal is amplified by the gain of another piece of equipment, both the
+noise and the signal from previous devices in the signal chain are amplified. For
+example, if your sound card is turned down and you turn the gain up on your mixer to compensate, the signal-to-noise
+ratio of the sound card output will be low and the mixer's gain will amplify
+both the signal and the noise from the sound card. The end result will have more
+noise than if the output of the sound card was turned up before the signal
+reached the mixer and the mixer's gain did not have to be turned up.
+
+.. note:: To adjust the output volume of a sound system while maintaing a
+             high signal-to-noise ratio, the gain should be
+             adjusted as close to the speakers as possible. Refer to
+             the :ref:`Gain Knob <interface-gain-knob>` section for details.
```

Comment:
The tern "gain" is confusing for me here. Here an attempt to improve it: 

To adjust the output volume of a sound system while maintaing a
high signal-to-noise ratio, use the preferably an analogue control 
as close to the speakers as possible. 
Refer to the :ref:`Gain Knob <interface-gain-knob>` section for details.